releases/release-1.12/release_team.md

```diff
@@ -1,13 +1,14 @@
 | **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
 | ------ | ------ | ------ |
-| Lead |||
-| Features | Stephen Augustus ([@justaugustus](https://github.com/justaugustus)) ||
-| CI Signal |||
-| Test Infra |||
-| Bug Triage |||
-| Branch Manager |||
-| Docs |||
-| Release Notes |||
-| Communications |||
-| Patch Release Manager |||
+| Lead | Tim Pepper ([@tpepper](https://github.com/tpepper)) | Aishwarya Sundar ([@AishSundar](https://github.com/AishSundar)) |
+| Features | Stephen Augustus ([@justaugustus](https://github.com/justaugustus)) | Kendrick Coleman ([@kacole2](https://github.com/kacole2)), Robert Sandoval ([@robertsandoval](https://github.com/robertsandoval)), Rajendar Talatam ([@rajendar38](https://github.com/rajendar38)) |
```

Comment:
Is it fine that we'll have multiple shadows for this role?

Not against it, just would like to clarify.

Reply:
Yes, we're trying to overstaff this release cycle as:
1. We're doing it much earlier than usual, so if prior commitments prevent people from participating, we will still be reasonably staffed.
2. More eyes on every role should provide better coverage and allow us to iterate faster on improvements.